src/data/queries/personProfile.ts

```diff
@@ -76,13 +76,14 @@ export const formatter: QueryFormatter<PersonProfileData, StaffProfile> = ({
     completeBiography: entity?.field_complete_biography?.uri || null,
     completeBiographyCreate: entity.field_complete_biography_create,
     photoAllowHiresDownload: entity.field_photo_allow_hires_download,
-    vamcOfficalName:
-      entity?.field_office?.field_vamc_system_official_name || null,
+    vamcTitle: entity?.field_office?.title || null,
     office: entity.field_office,
     menu: formattedMenu,
     administration: {
       id: entity.field_administration?.drupal_internal__tid || null,
       name: entity.field_administration?.name || null,
     },
+    lastUpdated:
+      entity.changed || entity.field_last_saved_by_an_editor || entity.created,
```

Comment:
This is the correct order for staff profile when compared to current prod. Changed here because I didn't want to break other types.